### DISCUSSION TITLE
TOC 마지막 목차 active 안되는 현상

### DISCUSSION THREAD

**Discussion by Bang322** [pronist/hELLO#154] [Q&A]
**환경 (버그가 발생한 환경을 적어주세요.):**

- 운영체제: windows
- 브라우저: chrome
- 버전: 107.0.5304.106(공식 빌드) (64비트)
- 스킨 버전: 3.6.4

**설명**
TOC 부분에서 목차가 몇 개든 상관없이 항상 마지막 목차가 active 되지 않습니다. (uk-active 클래스 추가되지 않음)
중간에 스킨을 바꿔서 뭔가 꼬인 게 있나 싶어서 새 글로 테스트를 해봐도 똑같은 현상입니다.

**발생 과정**
마지막 목차까지 스크롤하거나 TOC 부분에서 마지막 목차 클릭

**Comment by pronist** [comment]
@Bang322 안녕하세요.

제가 확인할 수 있도록 글 주소 남겨주세요. 일단 제 블로그에서는 마지막 목차도 active 되는 것으로 확인을 했기때문에 다른 부분을 살펴보아야 할 것 같습니다.
